cwms-data-api/src/main/java/cwms/cda/data/dto/stream/StreamLocation.java

```diff
@@ -35,30 +35,32 @@
 import cwms.cda.formatters.annotations.FormattableWith;
 import cwms.cda.formatters.json.JsonV1;
 
-import java.util.Objects;
-
-@FormattableWith(contentType = Formats.JSON, formatter = JsonV1.class)
+@FormattableWith(contentType = Formats.JSONV1, formatter = JsonV1.class, aliases = {Formats.DEFAULT, Formats.JSON})
 @JsonDeserialize(builder = StreamLocation.Builder.class)
 @JsonInclude(JsonInclude.Include.NON_NULL)
 @JsonNaming(PropertyNamingStrategies.KebabCaseStrategy.class)
 public final class StreamLocation implements CwmsDTOBase {
 
     private final LocationIdentifier streamLocationId;
-    private final StreamJunctionIdentifier streamJunctionIdentifier;
+    private final StreamNode streamNode; //the node representation of this location containing the stream id, bank, and station
     private final Double publishedStation;
     private final Double navigationStation;
     private final Double lowestMeasurableStage;
     private final Double totalDrainageArea;
     private final Double ungagedDrainageArea;
+    private final String areaUnit;
+    private final String stageUnit;
 
     private StreamLocation(Builder builder) {
-        this.streamJunctionIdentifier = builder.streamJunctionIdentifier;
-        this.streamLocationId = builder.streamLocationId;
+        this.streamNode = builder.streamNode;
+        this.streamLocationId = builder.streamId;
```

Comment:
Should keep the variable names consistent between the DTO and the builder.

Reply:
updated streamId to streamLocationId